.github/workflows/pr.yml

```diff
@@ -12,28 +12,25 @@ jobs:
         with:
           fetch-depth: 1
       - name: Set up JDK 8
-        uses: olafurpg/setup-scala@v11
+        uses: actions/setup-java@v3
```

Comment:
We need to switch the plugin here because of https://github.com/actions/setup-java#supported-distributions

> NOTE: AdoptOpenJDK got moved to Eclipse Temurin and won't be updated anymore. It is highly recommended to migrate workflows from adopt to temurin to keep receiving software and security updates. See more details in the [Good-bye AdoptOpenJDK post](https://blog.adoptopenjdk.net/2021/08/goodbye-adoptopenjdk-hello-adoptium/).